### PR TITLE
fix: Resources route

### LINKED ISSUE #261: [Bug]: Lenra apps does not seems to manage image in sub directory
### What happened?

In a Lenra app using the JavaScript template I tried to replace the default `logo.png` by a file in a sub-directory of `src/resources`.

When I expose the app and try to load the resource manually it works, but the app can't load it.

### What browsers are you seeing the problem on?

_No response_

### Version

?

### Relevant log output

```Shell
No logs in the app but there is this log in the devtool:


lenra-devtool-1  | 12:15:47.714 request_id=F19QLtP5w_4i5YwAAASB [info] GET /api/apps/00000000-0000-0000-0000-000000000000/resources/img/logo-characters.png
lenra-devtool-1  | 12:15:47.715 request_id=F19QLtP5w_4i5YwAAASB [info] Sent 404 in 221µs
```
```

### PR DESCRIPTION
**Pull request by pichoemr**
Closes https://github.com/lenra-io/lenra_cli/issues/261

## Description of the changes

Replace :ressource to *ressource to accept sub path

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
  - [ ] 👍 yes
  - [ ] 🙅 no, because they aren't needed
  - [x] 🙋 no, because I need help

https://github.com/lenra-io/server/pull/383

### I added/updated the documentation about my changes

- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by taorepoara** [comment]
Can you please add postgres to the docker-compose and document how to use it to run tests in the README as discussed before.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,23 @@
...
+You will first need to start two databases, postgres and mongo.
+
+command at the root of this project or running by hand the docker commands like so :
+* Start PostgreSQL with Docker `docker run --restart always -p 5432:5432 --name lenra-postgres -e POSTGRES_DB=lenra_dev -e POSTGRES_PASSWORD=postgres -d postgres`
+* Start MongoDB with Docker `docker compose up` or
+  - `docker run --restart always -p 27017:27017 --name lenra-mongo -e MONGO_INITDB_DATABASE=test -e CONFIG='{"_id" : "rs0", "members" : [{"_id" : 0,"host" : "mongodb:27017"}]}' -d mongo:5 mongod --replSet rs0`
+
+You will then need to install and setup elixir prerequisites for the application_runner to run properly :
+
+* Install erlang in version 24.2 and elixir in version 1.12.3 otp-24
```
Comment (taorepoara) [inline README.md]: Isn't the needed elixir version 1.13 ?

File: README.md [hunk 2]
```diff
@@ -50,6 +50,23 @@
 ### Prerequisites
 
 You can run Application Runner itself, but it is generally built as a library and needs a Phoenix parent application to access all functionalities: 
+
+You will first need to start two databases, postgres and mongo.
+
+command at the root of this project or running by hand the docker commands like so :
```
Comment (taorepoara) [inline README.md]: You should add the docker-compose way to start both databases. Do not remove the basic docker way, but let it in second place.

**Review by taorepoara — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -50,6 +50,23 @@
...
+* Start PostgreSQL with Docker `docker run --restart always -p 5432:5432 --name lenra-postgres -e POSTGRES_DB=lenra_dev -e POSTGRES_PASSWORD=postgres -d postgres`
+* Start MongoDB with Docker `docker compose up` or
+  - `docker run --restart always -p 27017:27017 --name lenra-mongo -e MONGO_INITDB_DATABASE=test -e CONFIG='{"_id" : "rs0", "members" : [{"_id" : 0,"host" : "mongodb:27017"}]}' -d mongo:5 mongod --replSet rs0`
+
+You will then need to install and setup elixir prerequisites for the application_runner to run properly :
+
+* Install erlang in version 24.2 and elixir in version 1.12.3 otp-24
+* Create the database and start migration `mix setup`. This is equivalent to running the following commands :
```
Comment (taorepoara) [inline README.md:64]: `mix setup` does not work. I have this issue: `** (Mix) The task "setup" could not be found`

File: README.md [hunk 4]
```diff
@@ -50,6 +50,23 @@
...
+  - `docker run --restart always -p 27017:27017 --name lenra-mongo -e MONGO_INITDB_DATABASE=test -e CONFIG='{"_id" : "rs0", "members" : [{"_id" : 0,"host" : "mongodb:27017"}]}' -d mongo:5 mongod --replSet rs0`
+
+You will then need to install and setup elixir prerequisites for the application_runner to run properly :
+
+* Install erlang in version 24.2 and elixir in version 1.12.3 otp-24
+* Create the database and start migration `mix setup`. This is equivalent to running the following commands : 
+  * `mix deps.get` to install the dependencies
+  * `mix ecto.create` to create database
+  * `mix ecto.migrate` to start all migration and have an up-to-date database
+  * `mix run priv/repo/seeds.exs` to fill database with default values
```
Comment (taorepoara) [inline README.md:68]: I don't think that there is seeds in this project

File: README.md (hunk 2 above)
Comment (pichoemr) [reply README.md]: In this project we don't have docker-compose to start both databases but only to start mongo, so I say in the mongo part that we can start it with docker-compose or with commands.

File: README.md (hunk 1 above)
Comment (pichoemr) [inline README.md]:
```suggestion
* Install erlang in version 24.2 and elixir in version 1.13 otp-24
```

File: README.md (hunk 2 above)
Comment (taorepoara) [reply README.md]: You should add postres in the docker-compose. It's easier to use

**Review by jonas-martinez — APPROVED** [review]
LGTM

**Review by taorepoara — APPROVED** [review]

**Comment by taorepoara** [comment]
Still needs documentation changes but it's not due to this PR so it can be handled later

File: README.md (hunk 3 above)
Comment (taorepoara) [reply README.md:64]: See issue: https://github.com/lenra-io/application-runner/issues/344

File: README.md (hunk 4 above)
Comment (taorepoara) [reply README.md:68]: See https://github.com/lenra-io/application-runner/issues/345

**Pull request merged**